### PR TITLE
i18n(ko-KR): update `astro-syntax.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

update `astro-syntax.mdx`

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by n0hack** [comment]
고생하셨습니다.
LGTM! ✨

**Comment by jsparkdev** [comment]
> 고생하셨습니다. LGTM! ✨

감사합니다 :)

File: src/content/docs/ko/core-concepts/astro-syntax.mdx [hunk 1]
```diff
@@ -62,7 +62,7 @@ function handleClick () {
 
 대신 순수 JavaScript처럼 클라이언트 측 스크립트를 사용하여 이벤트 핸들러를 추가하세요.
 
-```astro
+```astro title="do-this-instead.astro"
 ---
 // do-this-instead.astro
```
Comment (Yan-Thomas) [inline src/content/docs/ko/core-concepts/astro-syntax.mdx:67]:
```suggestion
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🙌

**Pull request merged**